R/datatypes-classes.R

```diff
@@ -280,3 +280,11 @@ datatype__min <- function() {
   self$`_dt`$min() |>
     wrap()
 }
+
+#' @param abbreviated Use short form of datatype name, used for instance in
+#' [`glimpse()`][dataframe__glimpse].
+#' @noRd
+datatype__as_str <- function(abbreviated = FALSE) {
+  self$`_dt`$as_str(abbreviated = abbreviated) |>
+    wrap()
+}
```

Comment:
Why do you need to add this?
(I do not want to add this. I was thinking that perhaps something like the S3 method of the format function needs to be added instead.)

Reply:
I don't understand the issue, it's simply here so that we can do `x$as_str()` instead of `x$`_dt`$as_str()`. Why is this a problem? I also don't see any S3 methods for `format()` currently so I don't think it should be added in this PR.

Reply:
In other words, I am not sure if the method `as_str` of dtype should be made public. (That's why it hasn't been added so far)

Reply:
So you'd rather have `format.polars_dtype`?

Reply:
Yes.
So just made that a TODO and now can't use `` x$`_dt`$as_str() ``?